BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -697,8 +697,8 @@ Cryptographic algorithms weaken over time as cryptanalysis improves: algorithms
 * Implementations SHOULD NOT negotiate cipher suites based on
                non-ephemeral (static) finite-field Diffie-Hellman key agreement. Similarly, implementations SHOULD NOT negotiate non-ephemeral elliptic curve DH key agreement.
 
-  Rationale: The former cipher suites, which have assigned values prefixed by "TLS_DH_*", have several drawbacks, especially
-               the fact that they do not support forward secrecy. The latter also lack forward secrecy, and are subject to invalid curve attacks {{Jager2015}}.
+  Rationale: The former cipher suites, which have assigned values prefixed by "TLS_DH_\*", have several drawbacks, especially
```

Comment:
The import of the backslash is not clear to me, but perhaps I don't understand the convention in this context...

Reply:
The HTML rendering was getting italicized for some reason (the combination of underscore and asterisk somehow), and the backslash fixed it.